### ISSUE TITLE
Remove crate version from docs.rs

### ISSUE THREAD

**Issue by sunjay** [rust-lang/docs.rs#158]
Is there a way to remove certain versions of a crate from docs.rs?

**Comment by onur** [comment]
Currently there isn't a way to remove crates from docs.rs. I am planning to add a method to check yanked crates and show a warning or remove the crate completely from docs.rs.

For now, I can manually remove the crate if there is an urgency.

**Comment by sunjay** [comment]
Could you remove the yanked versions of the turtle crate? That's currently all the versions. 

Thanks

**Comment by pravic** [comment]
+ futures 0.2.x have been yanked as well, see #221

**Comment by jyn514** [comment]
We've since redirected the docs for yanked crates for the latest version. It would be nice to add a badge saying that the crate has been yanked e.g. on https://docs.rs/crate/futures/0.2.3-docs-yank.4.

**Comment by jyn514** [comment]
Oops, I think that might have been done by `futures` in their docs actually. We have a long way to go for handling yanks :sweat_smile: 

See also #112, #322 , #561

**Comment by Nemo157** [comment]
@sunjay with #614 fixed so that the yanked status of 0.1.* is updated, do you still want the docs fully removed? The old versions show warnings that they are yanked and suggest redirecting to the prereleases.

**Comment by sunjay** [comment]
> @sunjay with #614 fixed so that the yanked status of 0.1.* is updated, do you still want the docs fully removed? 

Yes, that would be great. Those versions were for a different crate that used to use that name. Removing them will avoid some confusion so please do that. Thanks!

**Comment by jyn514** [comment]
We have a way to delete entire crates (`delete-crate`), but it sounds like you're asking for specific versions to be removed which is a little harder.

**Comment by sunjay** [comment]
Yes. I only want the 0.x versions removed. Back when this issue was created those were the only versions. :sweat_smile:

**Comment by pietroalbini** [comment]
I'm wondering if we should just hide the documentation of all yanked crates.

**Comment by jyn514** [comment]
Currently what we do is we don't show them unless you explicitly go to the link, and all the links to it have a big warning sign in addition to the 'this crate has been yanked' banner.

![image](https://user-images.githubusercontent.com/23638587/81220725-ca0b2c00-8faf-11ea-9e56-da430fc6593e.png)

![image](https://user-images.githubusercontent.com/23638587/81220798-e909be00-8faf-11ea-8035-ec25ab73f856.png)


 We could also hide the link to the yanked version if you like, but I'm starting to question how applicable this is to other use cases, it seems a little specific to `turtle`.

**Comment by sunjay** [comment]
Yeah I'd be fine with them just being deleted for the `turtle` crate. I can see why building something more general might cause unexpected issues.

**Comment by Kixiron** [comment]
I'm not necessarily against hiding (and potentially deleting, we need to do an audit of how much space is taken up by yanked crates) yanked docs, but it'll inevitably bring up the question/objection that we aren't supposed to be the ones who decide that docs aren't shown. Like I said, I'm not against it though, as I personally feel like someone who's using yanked crates can be reasonably expected to build docs locally and I think it'd get rid of what is basically a class of issues and considerations that have to come up so often

**Comment by jyn514** [comment]
I have deleted all the 0.x versions of turtle. The previous owner still shows up though (#900), that's waiting on #766.

**Comment by jyn514** [comment]
Sorry for the spam from rustbot, I've reached out to the infra team to fix it. In the meantime feel free to reach out to us on Discord: https://discord.gg/f7mTXPW

**Comment by pietroalbini** [comment]
I'm scared of commenting now.

**Comment by Mark-Simulacrum** [comment]
Even if we delete rustbot's messages github will steal leave a bunch of "message deleted" markers -- it might be easier to just open a new issue to track this. I'll leave it up to @jyn514 (or docs.rs team in general) though -- on triagebot's end things should be fixed now.

**Comment by jyn514** [comment]
I removed the old owner from turtle.

**Comment by sunjay** [comment]
Thanks for all your help @jyn514!
